tests/CMakeLists.txt

```diff
@@ -95,3 +104,4 @@ add_test(test_helpers test_helpers)
 add_test(test_crypto test_helpers)
 add_test(test_tx test_tx)
 add_test(test_web3 test_tx)
```

Comment:
aahh, this is a bug, it should be `add_test(test_web3 test_tx)` of course

Reply:
yes, and we have #121 for the same.